Automattic/jetpack
# Tools: Add excludelist cleanup, and use in CI

#### Changes proposed in this Pull Request:

I see we have four files in eslint-excludelist.json now that no longer have any lints. Let's make a tool to make it easier to keep the exclude lists up to date.

And while we're at it, we may as well run it in CI so people can be congratulated and prompted to do the cleanup themselves.

#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* See that the new test here passes.
* Make a PR that touches some excluded files (e.g. #18251), and see that it detects them properly too.

#### Proposed changelog entry for your changes:


* None needed, developer only.
